.circleci/config.yml

```diff
@@ -1,30 +1,34 @@
 version: 2
 jobs:
-  build:
+  no_pic:
     working_directory: ~/dmd
     docker:
       - image: circleci/node:4.8.2
     parallelism: 2
-    branches:
-      ignore:
-        - dmd-1.x
```

Comment:
This didn't work with the new workflow setting and AFAICT is only required for the respective branch anyways.